easybuild/framework/easyblock.py

```diff
@@ -3119,9 +3130,14 @@ def skip_step(self, step, skippable):
             self.log.info("Skipping %s step because of forced module-only mode", step)
             skip = True
 
+        elif sanity_check_only and step != SANITYCHECK_STEP:
+            self.log.info("Skipping %s step because of sanity-check-only mode", step)
+            skip = True
+
         else:
-            self.log.debug("Not skipping %s step (skippable: %s, skip: %s, skipsteps: %s, module_only: %s, force: %s",
-                           step, skippable, self.skip, self.cfg['skipsteps'], module_only, force)
+            msg = "Not skipping %s step (skippable: %s, skip: %s, skipsteps: %s, module_only: %s, force: %s, "
```

Comment:
just wondering, for the `msg +=` is there any style that's better, e.g. 
```
msg = ("Not skipping %s step (skippable: %s, skip: %s, skipsteps: %s, module_only: %s, force: %s, "
            "sanity_check_only: %s)")
```
or  using `''.join()` or something else?

Also there is a `)` missing in the string itself, behind the last `%s`.

Reply:
We use the `+=` approach all throughout the code, I think it's clearer sticking to that...